package.json

```diff
@@ -377,18 +377,16 @@
   },
   "dependencies": {
     "crypto": "^1.0.1",
-    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#0.3.0",
-    "opener": "^1.5.2",
-    "@microsoft/ads-service-downloader": "1.0.2",
+    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#1.3.2",
+    "@microsoft/ads-service-downloader": "^1.1.0",
+    "@microsoft/ads-extension-telemetry": "^1.3.3",
```

Comment:
I would suggest updating to 1.3.4 actually - there's a small fix I did to ensure that we don't throw an error on initialization either.

Reply:
done